@navikt/core/react/src/form/combobox/types.ts

```diff
@@ -91,13 +108,13 @@ export interface ComboboxProps
   /**
    * Callback function triggered whenever an option is selected or de-selected
    *
-   * @param option
+   * @param option - The option value
    * @param isSelected - Whether the option has been selected or unselected
    * @param isCustomOption - Whether the option comes from user input, instead of from the list
```

Comment:
Man trenger ikke ha strek mellom navn og beskrivelse
```suggestion
   * @param option The option value
   * @param isSelected Whether the option has been selected or unselected
   * @param isCustomOption Whether the option comes from user input, instead of from the list
```